packages/manager/src/components/HighlightedMarkdown/HighlightedMarkdown.tsx

```diff
@@ -82,7 +86,13 @@ export const HighlightedMarkdown = (props: HighlightedMarkdownProps) => {
 
   return (
     <Typography
-      className={classNames(classes.root, 'formatted-text', className)}
+      className={classNames(
+        {
+          [classes.root]: true,
+          'formatted-text': true,
+        },
+        className
+      )}
```

Comment:
I believe that this works the same and is more simple

```suggestion
      className={classNames(classes.root, 'formatted-text', className)}
```